src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1077,6 +1071,8 @@ public override void OnLightLevelChanged(float lightLevel)
         // is non-zero too.
         if (maxLightLevel > 0.0f && templateMaxLightLevel > 0.0f)
         {
+            var lightLevel = maxLightLevel * dayLightFraction;
+
             // Normalise by maximum light level in the patch
             camera!.LightLevel = lightLevel / maxLightLevel;
```

Comment:
Doesn't the math cancel out here.

You have first `ligthLevel = maxLightLevel * dayLightFraction` and then substituting that here gives:

```
lightLevel / maxLightLevel
maxLightLevel * dayLightFraction / maxLightLevel
dayLightFraction
```

So shouldn't this line just be `camera!.LightLevel = dayLightFraction;`?

Reply:
That's indeed right...